packages/xrpl/src/Wallet/defaultFaucets.ts

```diff
@@ -16,11 +16,16 @@ export enum FaucetNetwork {
   Devnet = 'faucet.devnet.rippletest.net',
 }
 
-export const FaucetNetworkPaths: Record<string, string> = {
+export const faucetNetworkPaths: Record<string, string> = {
   [FaucetNetwork.Testnet]: '/accounts',
   [FaucetNetwork.Devnet]: '/accounts',
 }
```

Comment:
Can we delete this map? All the faucets that I'm aware of use the `/accounts` endpoint. We can introduce this map if there is a change to that policy.

This is also an opportunity to simplify `getFaucetPath` method below.

Reply:
Historically, some faucets have been set up differently with different endpoints. All of Ripple's faucets may use the same setup, but faucets other people set up may not have the same endpoint.